### DISCUSSION TITLE
future ready talent

### DISCUSSION THREAD

**Discussion by amit457** [github/india#291] [GitHub Platform]
is anyone able to open the course provided in FRT portal because i am not able to open

**Comment by ArjunSrinivasanR** [comment]
I was also unable to open the drop-down menu if that is the problem that you are facing. I simply switched to Incognito tab and filled in all the details again and it worked. (Maybe clear history and try reloading but that didnt work for me)

**Comment by Kamaruddheen** [comment]
Yeah man, looks like some features are not working in **Google Chrome** except other browsers (Firefox, Edge) I guess so. I tried on **_Firefox and Edge browser_**. It is working fine 😊

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@amit457 
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@amit457 Are you able to open the course provided in FRT Portal?
If not please connect us here, we will assist you personally over voice or chat as you prefer 🤗 
Link to join: https://discord.gg/ceMXzhfaka

If this answers your question please mark this as answered!
**THANK YOU!**

**Comment by FRTMentor** [comment]
Please reach out our support team.
support@futurereadytalent.in
